provider/manifest/service.go

```diff
@@ -90,7 +89,7 @@ type service struct {
 	managers  map[string]*manager
 	managerch chan *manager
 
-	deploymentch chan runner.Result
+	// deploymentch chan runner.Result
```

Comment:
comment why it should stay commented?

Reply:
Its not used anywhere so I've commented it out.

Reply:
worth to remove then, so won't confuse in further